### ISSUE TITLE
Unknown word CssSyntaxError

### ISSUE THREAD

**Issue by Aviortheking** [ota-meshi/stylelint-plugin-stylus#24]
it seems this trigger the error in the title
```stylus
.icon
	vertical-align sub
	margin 2px
```
it happens on most of my files, the only files where this error do not appear on files that contains another error

_Sorry if i'm not in the correct repo_

**Comment by ota-meshi** [comment]
Thank you for this issue.
I can't reproduce it. Can you share a minimal repository to reproduce?

**Comment by Aviortheking** [comment]
here is it https://github.com/Aviortheking/stylelint-error-report

**Comment by ota-meshi** [comment]
You need to use the `--custom-syntax` option when running stylelint.

```bash
stylelint ... --custom-syntax stylelint-plugin-stylus/custom-syntax`
```

https://github.com/ota-meshi/stylelint-plugin-stylus#custom-syntax

**Comment by Aviortheking** [comment]
Thanks, I didn't thinks that I would still have to add the custom-syntax argument
